scripts/cli.py

```diff
@@ -247,6 +248,22 @@ def set_ext_proto_path(self, proto_path=None):
         else:
             logging.info('No KNoT Protocol path passed')
 
+    def set_ext_dts_ovl_path(self, dts_ovl_path=None):
+        """
+        In case of no DTS overlay path passed, keep it as None.
+        """
+        # Use default DTS files if none is passed
+        if dts_ovl_path is not None:
+            self.ext_dts_ovl_path = dts_ovl_path
+
+        # Log in case of a DTS overlay path passed
+        if self.ext_dts_ovl_path is not None:
```

Comment:
The only place where you set `self.ext_dts_ovl_path` is in the if above.
I think that this if's can be merged

Reply:
This if just makes sense on next commit

Reply:
Done.